index.html

```diff
@@ -3,176 +3,46 @@
-          <p>Renders an overlay with internal debug data, including playback state, timing metrics, and live player information.</p>
-        </div>
-      </div>
-    </a>
-  </li>
-  <li>
-    <a href="packages/user-preferences/index.html">
-      <div class="package-container">
-        <img src="assets/user-preferences.png" width="150">
-        <div>
-          <h4>user-preferences</h4>
-          <p>Stores and retrieves user settings like volume, playback rate, text track, and audio track using localStorage.</p>
-        </div>
-      </div>
-    </a>
-  </li>
-  <li>
-    <a href="packages/big-replay-button/index.html">
-      <div class="package-container">
-        <img src="assets/big-replay-button.png" width="150">
-        <div>
-          <h4>big-replay-button</h4>
-          <p>Displays a prominent replay button when playback ends, reusing the standard play button’s appearance and behavior.</p>
-        </div>
-      </div>
-    </a>
-  </li>
-  <li>
-    <a href="packages/brand-button/index.html">
-      <div class="package-container">
-        <img src="assets/brand-button.png" width="150">
-        <div>
-          <h4>brand-button</h4>
-          <p>Adds a configurable button to the control bar with a custom SVG icon and URL for linking to external destinations.</p>
-        </div>
-      </div>
-    </a>
-  </li>
-  <li>
-    <a href="packages/thumbnail-preview/index.html">
-      <div class="package-container">
-        <img src="assets/thumbnail-preview.png" width="150">
-        <div>
-          <h4>thumbnail-preview</h4>
-          <p>Displays timestamp-aligned thumbnails during timeline hover using a sprite sheet and optional scaling configuration.</p>
-        </div>
-      </div>
-    </a>
-  </li>
-  <li>
-    <a href="packages/card/index.html">
-      <div class="package-container">
-        <img src="assets/card.png" width="150">
-        <div>
-          <h4>card</h4>
-          <p>Displays a card component containing a title, duration, and a preview image.</p>
-        </div>
-      </div>
-    </a>
-  </li>
-</ul>
+<ul id="content"></ul>
+<script type="module">
+  import extensions from './assets/extensions.json';
+  import githubIcon from './assets/github-logo.svg?raw';
+
+  const ul = document.getElementById('content');
+
+  extensions.forEach(({ name, description }) => {
+    const li = document.createElement('li');
+
+    li.innerHTML = `
+    <a class="info-container"
+       title="Launch Demo"
+       href="packages/${name}/index.html"
+       target="_blank">
+      <img src="img/${name}.png" width="150">
+      <div>
+        <h4>${name}</h4>
+        <p>${description}</p>
+      </div>
+    </a>
+    <div class="link-container">
+      <a href="https://github.com/SRGSSR/pillarbox-web-suite/tree/main/packages/${name}#readme"
+         class="github-link"
+         title="See in Github"
+         target="_blank">
+        ${githubIcon}
+      </a>
+    </div>
+  `;
+
+    ul.appendChild(li);
+  });
```

Comment:
To avoid having multiple repaints we can use `document.createDocumentFragment`. Something like :

```javascript
// ...

const frag = document.createDocumentFragment();

 extensions.forEach(({ name, description }) => {
    const li = document.createElement('li');
    
    // all the magick ...

   frag.append(li);
});

// only one repaint
ul.append(frag);
```